src/stores/modelStore.ts

```diff
@@ -45,7 +45,7 @@ export class ComfyModelDef {
 
   constructor(name: string, directory: string) {
     this.name = name
-    this.title = name
+    this.title = name.replaceAll('\\', '/').split('/').pop()
```

Comment:
We should normalized the path in the backend like https://github.com/comfyanonymous/ComfyUI/pull/4906

Reply:
cc @comfyanonymous

Reply:
@mcmonkey4eva Can you make a PR to the main repo fixing that?

Reply:
that is a much larger change than you might realize. We should probably bring up a separate discussion on that first

Reply:
(This was discussed elsewhere as something to not change for now)